inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -336,15 +332,20 @@ void GNAModelSerial::Export(void * basePointer, size_t gnaGraphSize, std::ostrea
     writeBits(header, os);
 
     for (auto &name : inputNames) {
-        writeBits(name.size(), os);
+        ulong test = sizeof(size_t);
+        writeBits(sizeof(size_t), os);
+        writeBits(name.size() + 1, os);
         writeNBytes(name.c_str(), name.size(), os);
+        writeBits('\0', os);
```

Comment:
Instead, update the previous line
`writeNBytes(name.c_str(), strlen(name.c_str()) ` **+ 1** `, os);`